src/forms/propertiesdialog.ui

```diff
@@ -689,7 +689,17 @@
                 </property>
                </widget>
               </item>
-              <item row="11" column="0">
+              <item row="11" column="0" colspan="2">
+               <widget class="QCheckBox" name="openNewTabRightToActiveTab">
```

Comment:
In general we append widget type to their names. `openNewTabRightToActiveTab` -> `openNewTabRightToActiveTabCheckBox`

Reply:
Done in new revision of PR